Makefile

```diff
@@ -11,21 +11,23 @@
 USER = $(shell whoami)
 SYSTEMD = /etc/systemd/system
 BIN_PATH = /usr/local/bin
-CONF_PATH = /home/$(USER)/printer_data/config
+PRINTER_DATA_PATH=/home/$(USER)/printer_data
+CONF_PATH = $(PRINTER_DATA_PATH)/config
 
 all:
 	$(MAKE) help
 
 install: ## Install Spyglass as service
 	@printf "\nCopying systemd service file ...\n"
-	@sudo cp -f "${PWD}/resources/spyglass.service" $(SYSTEMD)
+	@sudo ln -sf "${PWD}/resources/spyglass.service" $(SYSTEMD)
 	@sudo sed -i "s/%USER%/$(USER)/g" $(SYSTEMD)/spyglass.service
 	@printf "\nCopying spyglass launch script ...\n"
 	@sudo ln -sf "${PWD}/scripts/spyglass" $(BIN_PATH)
 	@printf "\nCopying basic configuration file ...\n"
-	@cp -f "${PWD}/resources/spyglass.conf" $(CONF_PATH)
+	@ln -sf "${PWD}/resources/spyglass.conf" $(CONF_PATH)
```

Comment:
I'm not sure if it makes sense to create a symlink here. This way you would edit the config file inside your git repository and make it _Invalid_ I guess.

Reply:
Also would this overwrite the config with each update

Reply:
That was the idea, to auto update configs, so no changes would be need from user side, just clicking "update" in update manager it will be already done(because of symlink).

Reply:
So, in your pov it should be manual, and I should leave the "upgrade" to be executed by `make upgrade-moonraker`?

Reply:
The thing is that you would overwrite settings like resolution, fps, autofocus and so on.
Maybe I'm wrong and it would just automerge it?

Reply:
No, you're correct. Make sense 🤦